Assignment2/Assignment2_QNo1/SolAssignmentNo2_QNo1.rb

```diff
@@ -0,0 +1,32 @@
+# => 1. Write a class and define a class method which will return
+# => the object count of that class.
+# => Define the class
+class ObjectCnt
```

Comment:
@HrisikeshSinghaChowdhury Please use proper and meaningful class name something like ExampleClass  and some related name , ObjectCnt can not be a class name